packages/server/src/index.ts

```diff
@@ -84,7 +96,13 @@ export const start = async (
 
   registerEndpoints(app, ws, searchDir);
 
-  app.get("/*", (_, response) => {
+  app.use(Express.static(outDir));
+
+  app.get("/*", async (req, response) => {
+    const publicPath = await getPublicPath(searchDir, req.originalUrl);
```

Comment:
What if there's a query string attached? Perhaps we just want `req.path` (or more likely, `req.path.substring(1)`)?